packages/code-du-travail-frontend/pages/outils/[slug].js

```diff
@@ -57,12 +57,12 @@ const getSimulator = function(name) {
     default:
       return {
         title: "Outil introuvable",
-        component: OutilIntrouvable
+        component: ToolNotFound
```

Comment:
this looks a lot an object map {[slug]: props} we should only have a mapping between slug and components, the other data should come from the json  / Es
Another solution could be to create for each tools its dedicated page (using its slug) to get rid of that mapping